apps/site/lib/site_web/views/helpers/cms_router_helpers.ex

```diff
@@ -85,7 +85,7 @@ defmodule SiteWeb.CmsRouterHelpers do
   end
 
   @spec project_path(
-          Conn.t(),
+          Conn.t() | any,
```

Comment:
Why `any`? Is it not possible to be more specific about the type here? Seems like it would be desirable to do so if possible.

Reply:
I couldn't find a type for SiteWeb.Endpoint. I did some internet searching and didn't find anything helpful either.

Reply:
`module`